src/httpd_streaming.c

```diff
@@ -52,6 +52,7 @@ extern struct event_base *evbase_httpd;
 #define STREAMING_MP3_SAMPLE_RATE 44100
 #define STREAMING_MP3_BPS         16
 #define STREAMING_MP3_CHANNELS    2
+static int streaming_mp3_bitrate = 192000;
```

Comment:
I propose you rename the current streaming_quality to streaming_quality_in and make a new streaming_quality_out. The latter is then by default initialised with STREAMING_MP3_ (also a STREAMING_MP3_BIT_RATE), but all the defaults can then be changed via config.